src/components/withDemoChat.js

```diff
@@ -5,10 +5,10 @@ import {
 } from '@automattic/big-sky-agents';
 
 const withDemoChat = (ChildComponent) => {
-	const WithDemoChat = ({ apiKey, ...props }) => (
+	const WithDemoChat = ({ apiKey, feature, ...props }) => (
 		<ChatProvider
 			apiKey={apiKey}
-			feature="big-sky"
+			feature={feature}
```

Comment:
this seems to be working in terms of setting up a separate store slice but all the chats submitted still seem to go back to the tiny agent, even when the active agent has been changed 🤔